android/src/lib.rs

```diff
@@ -33,8 +41,48 @@ pub fn android_main() {
 }
 
 #[no_mangle]
-pub extern "system" fn Java_org_maplibre_1rs_MapLibreRs_android_1main(env: JNIEnv, class: JClass) {
+pub extern "system" fn Java_org_maplibre_1rs_MapLibreRs_android_1main(
+    env: JNIEnv,
+    class: JClass,
+    surface: JObject,
+) {
+    env_logger::init_from_env(env_logger::Env::default().default_filter_or("info"));
+
     let tag = CString::new("maplibre").unwrap();
     let message = CString::new("maplibre WOORKING").unwrap();
     ndk_glue::android_log(Level::Warn, &tag, &message);
+
+    unsafe {
+        let mut logpipe: [RawFd; 2] = Default::default();
+        libc::pipe(logpipe.as_mut_ptr());
+        libc::dup2(logpipe[1], libc::STDOUT_FILENO);
+        libc::dup2(logpipe[1], libc::STDERR_FILENO);
+        thread::spawn(move || {
+            let tag = CStr::from_bytes_with_nul(b"MapLibreStderr\0").unwrap();
+            let file = File::from_raw_fd(logpipe[0]);
+            let mut reader = BufReader::new(file);
+            let mut buffer = String::new();
+            loop {
+                buffer.clear();
+                if let Ok(len) = reader.read_line(&mut buffer) {
+                    if len == 0 {
+                        break;
+                    } else if let Ok(msg) = CString::new(buffer.clone()) {
+                        ndk_glue::android_log(Level::Info, tag, &msg);
+                    }
+                }
+            }
+        });
+    }
```

Comment:
CC https://github.com/rust-windowing/android-ndk-rs/issues/281, perhaps we should move this out of `ndk_glue` and make it generic :)

Perhaps some unification with the `android_logger` crate too.